msg/vehicle_constraints.msg

```diff
@@ -10,6 +10,7 @@ float32 speed_down	# in meters/sec
 float32 tilt    # in radians [0, PI]
 float32 min_distance_to_ground # in meters
 float32 max_distance_to_ground # in meters
+float32[4] velocity_limits    # maximum velocities determined by range sensor measurements [x positive, y positive, x negative, y negative] direction.
```

Comment:
Suggestion - consider splitting this into separate arrays for velocity min (x & y) and velocity max (x & y). Then in the code you can directly create a Vector2f.